tools/jxltran.cc

```diff
@@ -26,6 +29,11 @@ struct Args {
 
     cmdline->AddHelpText("\nFile format options:", 0);
 
+    cmdline->AddOptionFlag('\0', "pack",
```

Comment:
This is OK for now, but perhaps a more convenient interface would be to specify whether or not the output should use the container format, regardless of the input. The way it is now, you first need to know which format the input uses and then apply an action or not.

Reply:
I'm not planning to change that soon but I'll try to keep it in mind